### DISCUSSION TITLE
"WINDIR" environment variable pass through

### DISCUSSION THREAD

**Discussion by dmgass** [tox-dev/tox#2129] [General]
When we upgraded from 3.23.1 to 3.24.0 we noticed we had to configure tox to pass through the "WINDIR" environment variable when using 3.24.0 (too keep imports of matplotlib happy). I don't see anything related to this in the release notes for 3.24, nor do I see anything that sticks out in the code diffs. Does anyone know of a change in tox that could have caused this or should I be looking for other changes in our CI/CD system / environment that would explain it?

**Comment by jugmac00** [comment]
I also had a look and I do not see any relevant code changes.

Can you reproduce the problem locally? And a successful run, too?

When I have a look at our documentation, `WINDIR` does not show up at all:
https://tox.readthedocs.io/en/latest/config.html?highlight=environment#conf-passenv

**Reply by dmgass** [reply]
thanks for double checking the diffs for me ... I'll try isolating things on this end and get back to you if I see anything that points to tox
